pixi.toml

```diff
@@ -23,20 +32,50 @@ rattler-build-conda-compat = ">=1.2.0,<2.0.0a0"
 [feature.linux]
 # The linux feature runs on every platform that can run Docker.
 platforms = ["linux-64", "linux-aarch64", "osx-64", "osx-arm64", "win-64"]
-[feature.linux.tasks]
-build-linux = { cmd = "python build-locally.py", env = { BUILD_LOCALLY_FILTER = "linux*" } }
+[feature.linux.tasks.build-linux]
+description = "build for Linux inside a docker container"
+cmd = "python build-locally.py"
+env = { BUILD_LOCALLY_FILTER = "linux*" }
 
 # The osx env will run conda-build directly on the machine.
 [feature.osx]
 platforms = ["osx-64", "osx-arm64"]
-[feature.osx.tasks]
-build-osx = { cmd = "python build-locally.py", env = { BUILD_LOCALLY_FILTER = "osx*", OSX_SDK_DIR = "$PIXI_PROJECT_ROOT/.pixi/macOS-SDKs", MINIFORGE_HOME = "$CONDA_PREFIX", CONDA_BLD_PATH = "$PIXI_PROJECT_ROOT/build_artifacts" } }
+[feature.osx.tasks.build-osx]
+description = "build directly on a MacOS host machine"
+cmd = "python build-locally.py"
+[feature.osx.tasks.build-osx.env]
+BUILD_LOCALLY_FILTER = "osx*"
+CONDA_BLD_PATH = "$PIXI_PROJECT_ROOT/build_artifacts"
+MINIFORGE_HOME = "$CONDA_PREFIX"
+OSX_SDK_DIR = "$PIXI_PROJECT_ROOT/.pixi/macOS-SDKs"
 
 # The windows env will run conda-build directly on the machine.
 [feature.win]
-platforms = ["win-64"] # , "win-arm64"]
-[feature.win.tasks]
-build-win = { cmd = "python build-locally.py", env = { BUILD_LOCALLY_FILTER = "win*", MINIFORGE_HOME = "$CONDA_PREFIX", CONDA_BLD_PATH = "$PIXI_PROJECT_ROOT/build_artifacts" } }
+platforms = ["win-64"]
+[feature.win.tasks.build-win]
+description = "build directly on a Windows host machine"
+cmd = "python build-locally.py"
+[feature.win.tasks.build-win.env]
+BUILD_LOCALLY_FILTER = "win*"
+CONDA_BLD_PATH = "$PIXI_PROJECT_ROOT/build_artifacts"
+MINIFORGE_HOME = "$CONDA_PREFIX"
+
+[feature.grayskull.dependencies]
+grayskull = ">=2.7.3"
+[feature.grayskull.tasks.pypi]
+description = "generate a recipe for a PyPI package name with `grayskull`"
+cmd = "cd recipes && grayskull pypi --strict-conda-forge"
```

Comment:
we could also add `--recursive` here...

Reply:
Folks can do `pixi run pypi --recursive ...` themselves, right? I think it's fine like this.